package.json

```diff
@@ -27,6 +27,7 @@
     "redux-persist": "^5.10.0",
     "redux-thunk": "^2.3.0",
     "reselect": "^3.0.1",
+    "rrule": "^2.6.3",
```

Comment:
No need to import rrule, you can just make it "rrule like" I'll explain more below.